plugins/woocommerce-admin/client/shipping/test/shipping-recommendations-wrapper.test.tsx

```diff
@@ -46,7 +46,7 @@ describe( 'ShippingRecommendations', () => {
 			/>
 		);
 
-		expect( queryByText( 'Woocommerce Shipping' ) ).not.toBeInTheDocument();
+		expect( queryByText( 'WoCcommerce Shipping' ) ).not.toBeInTheDocument();
```

Comment:
There seems to be a few of these in this file:

```suggestion
		expect( queryByText( 'WooCommerce Shipping' ) ).not.toBeInTheDocument();
```

Reply:
I really do need a new keyboard and eyes 🥺